src/interactivity/win32/Clipboard.cpp

```diff
@@ -319,6 +300,35 @@ void Clipboard::CopyTextToSystemClipboard(const TextBuffer::TextAndColor& rows,
     globalHandle.release();
 }
 
+void Clipboard::CopyToSystemClipboard(std::string s, LPCWSTR lpszFormat)

```

Comment:
Can we rename the parameter to be something more descriptive than `s`? Adding the function description will also help this issue.

Reply:
Done.